test/Analysis/interval_analysis/edwards2montgomery.llzk

```diff
@@ -335,3 +336,18 @@ module attributes {veridise.lang = "llzk"} {
 // CHECK-NEXT:     %arg0[@v][@$super] in Entire
 // CHECK-NEXT:     %arg0[@v][@reciprocal] in Entire
 // CHECK-NEXT: }
+
+// CMP-LABEL: @Div StructIntervals {
+// CMP-NEXT:     compute {
+// CMP-NEXT:         %arg0 in Entire
+// CMP-NEXT:         %arg1 in TypeC:[ 1, 2013265920 ]
+// CMP-NEXT:         %self[@$super] in Entire
```

Comment:
I would like to see more LLZK tests that exercise different operations in the compute. The PR is quite large so more tests would be great here like bitwise ops, boolean ops, control flow, etc.